package-lock.json

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
is this really needed? If yes, please move alongside with the code.

Reply:
Not needed

Reply:
Please remove the file then.

Reply:
This still needs to be removed.